study.md

```diff
@@ -27,17 +27,22 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+It is not html. It is not javascript.
+
+https://teamtreehouse.com/community/isnt-the-dom-made-from-html
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+It is the DOM. It looks like Html but it changes as you move through the page and javascript manipulates it.
+
+https://css-tricks.com/dom/
 ```
```

Comment:
So this is a tricky one. The answer is kind of. It is NOT the DOM, but what we see in Dev Tools is a visual representation of the DOM.